docs/guides/stonesense-art-guide.rst

```diff
@@ -0,0 +1,63 @@
+.. _stonesense-art-guide:
+
+Stonesense art creation guide
+=============================
+
+Understanding sprites
+---------------------
+Understanding how Stonesense deals with sprites is central to anyone who wishes to modify the content.
```

Comment:
This section should *start* with a definition of what sprites are, not end with it.